src/audit-log/src/lib.rs

```diff
@@ -230,6 +232,17 @@ pub struct DropClusterReplicaV1 {
     pub replica_name: String,
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize, PartialOrd, PartialEq, Eq, Ord, Hash, Arbitrary)]
+pub struct DropClusterReplicaV2 {
+    pub cluster_id: String,
+    pub cluster_name: String,
+    // Events that predate v0.32.0 will not have this field set.
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
I copy-pasted this from V1, but I guess it's no longer relevant here. Should I remove this?